tests/test_matching_classes.py

```diff
@@ -0,0 +1,216 @@
+import pytest
+import rdflib
+from rdflib.namespace import OWL, RDF, RDFS
+from rdflib import Graph, URIRef
+import json
+import brickschema
+import sys
+
+sys.path.append("..")
+
+from bricksrc.version import BRICK_VERSION  # noqa: E402
+from bricksrc.namespaces import BRICK  # noqa: E402
+
+
+def getDict(g, q):
+    dict = {}
+
+    res = g.query(q)
+
+    for row in res:
+        c1 = row.c.toPython().replace("https://brickschema.org/schema/Brick#", "")
+        if row.p:
+            c2 = row.p.toPython().replace("https://brickschema.org/schema/Brick#", "")
+            if c1 in dict:
+                dict[c1].append(c2)
+            else:
+                dict[c1] = [c2]
+
```

Comment:
This logic can be replaced with a `defaultdict(list)` from the `collections` module

```python
from collections import defaultdict

d = defaultdict(list)

    res = g.query(q)

    for row in res:
        c1 = row.c.toPython().replace("https://brickschema.org/schema/Brick#", "")
        if row.p:
            c2 = row.p.toPython().replace("https://brickschema.org/schema/Brick#", "")
            d[c1].append(c2)

Reply:
Done